apps/deploy-web/tests/fixture/context-with-extension.ts

```
@@ -18,7 +18,8 @@ export const test = baseTest.extend<{
   extensionId: string;
   extPage: Page;
 }>({
-  context: async ({ headless }, use) => {
+  // eslint-disable-next-line no-empty-pattern
+  context: async ({}, use) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)